addons/tmf_loadouts/loadouts/cz_army_2020.hpp

```diff
@@ -0,0 +1,628 @@
+		"ItemGPS",
+		"ItemRadio"
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_ACRPara_m95"};
+	secondaryWeapon[] = {"ace_csw_staticM2ShieldCarry"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_ACRPara_m95"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripod"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_ACRPara_m95"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripodLow"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_ACRPara_m95"};
+	secondaryWeapon[] = {"CUP_launch_MAAWS"};
+	secondaryAttachments[] = {"cup_optic_maaws_scope"};
+	magazines[] += {
+		LIST_2("CUP_MAAWS_HEAT_M"),
+		"CUP_MAAWS_HEDP_M"
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CUP_B_ACRPara_m95"};
+	backpackItems[] = {
+		LIST_2("CUP_MAAWS_HEAT_M"),
+		"CUP_MAAWS_HEDP_M"
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"CUP_B_ACRPara_m95"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_MAAWS_HEAT_M"),
+		"CUP_MAAWS_HEDP_M"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+	magazines[] = {
+		LIST_9("CUP_30Rnd_556x45_CZ805"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"B_Carryall_wdl_F"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"B_Carryall_wdl_F"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {"B_Carryall_wdl_F"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_9K32Strela"};
```

Comment:
Replace with either Stinger or Igla. In reality they'd be using the RBS-70 in this role.